UT4MasterServer.Web/src/types/account.ts

```diff
@@ -1,5 +1,25 @@
+import { AccountFlag } from "@/enums/account-flag";
+
 export interface IAccount {
     id: string;
     displayName: string;
     email?: string;
+    name?: string;
+    failedLoginAttempts?: number;
+    lastLogin?: string;
+    numberOfDisplayNameChanges?: number;
+    ageGroup?: string;
+    headless?: boolean;
+    country?: string;
+    lastName?: string;
+    preferredLanguage?: string;
+    canUpdateDisplayName?: boolean;
+    tfaEnabled?: boolean;
+    emailVerified?: boolean;
+    minorVerified?: boolean;
+    minorExpected?: boolean;
+    minorStatus?: string;
+    cabinedMode?: boolean;
+    hasHashedEmail?: boolean
+    flags?: AccountFlag[];

```

Comment:
`flags` should not be returned in the standard account fetching endpoint, so i assume this shouldn't be here as well.

Reply:
Yeah I'm gonna have to move the whole thing to a different controller action, that's fine. I'd rather avoid using any of the epic endpoints so I don't have to worry about changing them

Reply:
We should maybe mark the ones used by the game so we know not to change em